articles/architecture-scenarios/b2c/branding.md

```diff
@@ -0,0 +1,121 @@
+
+* How will users know that they're still on your domain when they see your login page?
+
+* What do you need to do to provide additional browser security (e.g., implement Extended Validation)?
+
+* How will you direct users in the event of errors?
+
+### Universal Login and Login Pages
+
+The [Universal Login](/universal-login) process is the recommended method for authenticating users, and it centers around the Login Page. You can customize the Login Page via the Dashboard to support your organization's branding requirements. You can make [simple changes](/universal-login#simple-customization) that include changes to the logo, primary color, or background color, or you can make more [advanced changes](/universal-login#advanced-customization) that require you to directly modify the script powering the page itself.
+
+If you opt to customize the Login Page script, we strongly recommend that you:
+
+* [Use version control](/universal-login/version-control) to maintain prior versions
+* Deploy changes to your script via [automated deployments](#)
+
+Once modified, you are responsible for maintaining the script powering your Login Page, including updating the version number of the Lock widget used.
+
+::: warning
+Exercise caution regarding the use of third-party JavaScript on your Login Page, since sensitive security-related information often flows through the page.
+:::
+
+The Login Page comes with [multi-language support](/libraries/lock/v11/i18n) provided by default. In our experience, almost all applications need internationalization/localization, and while it is fairly simple to add during the development phases, it can become difficult at a later stage. As such, we recommend, at the very least, accounting for this need if you do not implement either feature during this phase.
+
+#### Custom login pages
+
+For those who want to build and use a fully-customized user interface using technologies like React or Angular.js, the Auth0 JavaScript SDK ([auth0js](/libraries/auth0js)) allows you to add Auth0 functionality to your custom login page.
+
+### Change Password
+
+The Change Password page is used whenever a user takes advantage of the [Password Reset](#) functionality. Like the Login Page, you can customize the Change Password page to reflect your organization's branding. 
+
+::: panel Content Delivery Networks
+If you're using custom Login and/or Change Password pages, we recommend using an external Content Delivery Network (CDN) to help speed up page load and browser rendering time for your pages.
+:::
+
+### Errors
+
+If there are issues with user sign up or login, Auth0 provides error messages that indicate what the problem is under the hood. However, the default messages are somewhat cryptic, especially to the end user, since it will likely be missing context that only you can supply. As such, we recommend customizing your error pages to provide the missing context-specific information directly to your users.
```

Comment:
```suggestion
If there are issues with user sign up or login Auth0 provides error messages that indicate what the problem is under the hood. The default messages however are somewhat cryptic, especially to the end user, since they will likely be missing context that only you can supply. As such, we recommend customizing your error pages to provide the missing context-specific information directly to your users.
```